doc/compiler/jitserver/Usage.md

```diff
@@ -20,7 +20,7 @@ OpenJDK Assembly Exception [2].
 SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 -->
 
-JITServer adds two additional *personas* to the JVM: client mode and server mode. To start the JVM in server mode, run the `jitserver` launcher present under `bin` directory (along side `java` launcher). To start the JVM in client mode, there is a new command line option: `-XX:+UseJITServer`
+JITServer adds two additional *personas* to the JVM: client mode and server mode. To start the JVM in server mode, run the `jitserver` launcher present under `bin` directory (along side `java` launcher)(The `bin` directory exists in `build/linux-x86_64-normal-server-release/images/jdk/bin` run `./jitserver` there to launch the JVM). To start the JVM in client mode, there is a new command line option: `-XX:+UseJITServer` (under bin run `./java -XX:+UseJITServer <application name>`)
```

Comment:
I would prefer to eliminate: `(The `bin` directory exists in `build/linux-x86_64-normal-server-release/images/jdk/bin` run `./jitserver` there to launch the JVM). `
because it refers to a directory structure created for build. We should be more generic on how to launch the jitserver